packages/appium/lib/extension/manifest.js

```diff
@@ -39,44 +39,49 @@ const INITIAL_MANIFEST_DATA = Object.freeze({
  * Given a `package.json` return `true` if it represents an Appium Extension (either a driver or plugin).
  *
  * The `package.json` must have an `appium` property which is an object.
- * @param {any} value
- * @returns {value is ExtPackageJson<ExtensionType>}
  */
-function isExtension(value) {
-  return (
+const isExtension = _.memoize(
```

Comment:
I don't see much point in memoizing this method. It's very simple

Reply:
yeah, probably not.  I'm going to make this a little more simple anyway